tests/Functional/ProxyGeneratorTest.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+namespace Zenstruck\Foundry\Tests\Functional;
+
+use Zenstruck\Foundry\AnonymousFactory;
+use Zenstruck\Foundry\Tests\Fixtures\Entity\Contact;
+use Zenstruck\Foundry\Tests\Fixtures\Entity\Post;
+use Zenstruck\Foundry\Tests\Fixtures\Factories\PostFactoryWithProxyGenerator;
+
+/**
+ * @author Wouter de Jong <wouter@wouterj.nl>
+ */
+final class ProxyGeneratorTest extends ProxyTest
+{
+    protected static $POST_FACTORY = PostFactoryWithProxyGenerator::class;
+
+    /**
+     * @test
+     */
+    public function can_assert_persisted(): void
+    {
+        $this->markTestSkipped('not supported');
+    }
+
+    /**
+     * @test
+     */
+    public function can_remove_and_assert_not_persisted(): void
+    {
+        $this->markTestSkipped('not supported');
+    }
+
+    /**
+     * @test
+     */
+    public function can_refetch_object_if_object_manager_has_been_cleared(): void
+    {
+        $this->markTestSkipped('not supported');
+    }
+
+    /**
+     * @test
+     */
+    public function exception_thrown_if_trying_to_refresh_deleted_object(): void
+    {
+        $this->markTestSkipped('not supported');
+    }
+
+    /**
+     * @test
+     */
+    public function can_force_set_and_save(): void
+    {
+        $this->markTestSkipped('not supported');
+    }
```

Comment:
I did not move the `forceSet`/`forceGet` methods to the factory (yet). Is this a feature we really want to keep in Foundry?

Reply:
I do find them useful for feature-rich models with no public access to certain fields (ie verification or password reset token).

I'd be fine with leaving them on the `Instantiator` or adding some helper functions.